cloud: add username prefix (#9755)
pingcap/docs: tidb-cloud/select-cluster-tier.md

The one-year free trial begins the day the first Developer Tier cluster is created.

### User name prefix

<!--Important: Do not update the section name "User name prefix" because this section is referenced by the TiDB error messages.-->

For each Developer Tier TiDB cluster, TiDB Cloud generates a unique prefix to distinguish it from others.

When you connect to a Developer Tier TiDB cluster via standard connection or when you create a database user, you must include the prefix in the user name and wrap the name with quotation marks.

> Take the default `root` user an example. If the prefix for your cluster is `3pTAoNNegb47Uc8`, you need to use `'3pTAoNNegb47Uc8.root'` to connect to your TiDB cluster.

Reviewer: For example, if the prefix for your cluster is `3pTAoNNegb47Uc8` and you must create a user with such a prefix, (eg:`3pTAoNNegb47Uc8.test`) and connect to your database with the username which has the prefix.

For example, your must connect to your cluster with username `3pTAoNNegb47Uc8.root`:

```shell
mysql --connect-timeout 15 -u '3pTAoNNegb47Uc8.root' -h gateway01.us-east-1.staging.shared.aws.tidbcloud.com -P 4000 -D test -p
```

When you create a new database user, you must create a user which has the same prefix:
```mysql
CREATE USER `3pTAoNNegb47Uc8. jeffrey`;
```

> For example, if the prefix for your cluster is `3pTAoNNegb47Uc8` and your database user name is `test`, you need to use `'3pTAoNNegb47Uc8.test'` to connect to your TiDB cluster.

Reviewer: 这里不太对。实际上用户名就是`3pTAoNNegb47Uc8.test`

Reviewer: ```suggestion
For example, if the prefix for your cluster is `3pTAoNNegb47Uc8` and you must create a user with such a prefix, (eg:`3pTAoNNegb47Uc8.test`) and connect to your database with the username which has prefix.

For example, your must connect to your cluster with username `3pTAoNNegb47Uc8.root`:

```shell
mysql --connect-timeout 15 -u '3pTAoNNegb47Uc8.root' -h gateway01.us-east-1.staging.shared.aws.tidbcloud.com -P 4000 -D test -p
```

When you create a new database user, you must create a user which has the same prefix:
```mysql
CREATE USER `3pTAoNNegb47Uc8. jeffrey`;
```

```

The one-year free trial begins the day the first Developer Tier cluster is created.

### User name prefix

<!--Important: Do not update the section name "User name prefix" because this section is referenced by TiDB backend error messages.-->

> For each Developer Tier cluster, TiDB Cloud generates a unique prefix to distinguish it from other clusters. For example, `3pTAoNNegb47Uc8`.
> 
> Whenever you use or create a database user name, you must include the prefix in the user name. For example:

Reviewer: ```suggestion
For each Developer Tier cluster, TiDB Cloud generates a unique prefix to distinguish it from other clusters.

Whenever you use or create a database user name, you must include the prefix in the user name. For example, let's say the prefix of your cluster is `3pTAoNNegb47Uc8`:
```

> Whenever you use or create a database user name, you must include the prefix in the user name. For example, assume that the prefix of your cluster is `3pTAoNNegb47Uc8`:

Reviewer 2: ```suggestion
Whenever you use or create a database user, you must include the prefix in the user name. For example, assume that the prefix of your cluster is `3pTAoNNegb47Uc8`:
```

> - To connect to your cluster

Reviewer 2: ```suggestion
- To connect to your cluster:
```

    ```shell

>     mysql --connect-timeout 15 -u '3pTAoNNegb47Uc8.root' -h gateway01.ap-northeast-1.prod.aws.tidbcloud.com -P 4000 -D test -p

Reviewer: ```suggestion
    mysql --connect-timeout 15 -u '3pTAoNNegb47Uc8.root' -h <host> -P 4000 -D test -p
```

    ```

> - To create a database user

Reviewer 2: ```suggestion
- To create a database user:
```

Reviewer: the database user is not `jefferey`, it's `3pTAoNNegb47Uc8.jeffrey`

>     ```mysql

Reviewer 2: ```suggestion
    ```sql
```

    CREATE USER '3pTAoNNegb47Uc8.jeffrey';
    ```

To get the prefix for your cluster, take the following steps:

> 1. In the TiDB Cloud console, navigate to the **Active Clusters** page and click the name of your cluster.

Author: ```suggestion
1. In the TiDB Cloud console, navigate to the **Active Clusters** page of your project and click the name of your cluster.
```

2. In the cluster information pane on the left, click **Connect**. The **Connect to TiDB** dialog is displayed.

> 3. In the dialog, locate **Step 2: Connect with a SQL client** and get the prefix.

Reviewer 2: I want to know how to get my user name prefix first and then use the preix to connect to my cluster.

Author: In **Step 2: Connect with a SQL client**, there will be a note that shows the prefix.
